examples/research_projects/pytorch_xla/README_xla.md

```diff
@@ -0,0 +1,126 @@
+# Stable Diffusion text-to-image fine-tuning using PyTorch/XLA
+
+The `train_text_to_image_xla.py` script shows how to fine-tune stable diffusion model on TPU devices using PyTorch/XLA.
+
+It has been tested on v4 and v5p TPU versions.
```

Comment:
can you be a little more specific on the TPU models you used? It would be nice if someone wants to reproduce this to know what is the amount of TPUs required to do this without hitting an OOM.

Reply:
I've added this to the sheet. Please review.